lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -1653,8 +1653,8 @@ def cmd_db_nmap_tabs(str, words)
   # Database management
   #
   def db_check_driver
-    if(not framework.db.driver)
-      print_error("No database driver installed. Try 'gem install pg'")
```

Comment:
Why was the suggestion to install the pg gem removed here?

Reply:
I believe the suggestion to `gem install pg` is not relevant given we now use `bundler`. Is this not correct @busterb?